src/bindgen.h

```diff
@@ -29,10 +29,17 @@
 #include "stdint.h"
 #include "stdbool.h"
 
+// types from detect.h made opaque
+typedef struct DetectEngineCtx DetectEngineCtx;
+typedef struct Signature Signature;
+typedef struct SigMatch SigMatch;
```

Comment:
The struct names for all these have a trailing `_`. If they exist in the same compilation unit as the real definitions, the compile will fail, so they should match.

Reply:
ok will fix